cli/src/generate/templates/js-binding.cc

```diff
@@ -4,9 +4,15 @@ typedef struct TSLanguage TSLanguage;
 
 extern "C" TSLanguage *tree_sitter_PARSER_NAME();
 
+const napi_type_tag LANGUAGE_TYPE_TAG = {
+  0x95840BEBF71E4E90, 0x9DC9419B874C0271
```

Comment:
But what are these magic numbers?

Reply:
Does it have to be a UUID? It'd be nice if it could represent tree-sitter:
```bash
b2sum -l64 <(printf tree) <(printf sitter) | \
  awk '{printf "0x" toupper($1) (NR == 1 ? ", " : "\n")}'
# 0x7FA28BFC1966AC2D, 0xA24BD16E50AD8E52
```
In any case, there should be a small explanation in a comment.

Reply:
Dunno. They specified a UUID... But I don't think anything stops you from using any magic number. You can probably even hex encode tree-sitter or something...